helix-view/src/editor.rs

```diff
@@ -1323,10 +1332,10 @@ impl Editor {
             }
             doc.set_version_control_head(self.diff_providers.get_current_head_name(&path));
 
-            let id = self.new_document(doc);
-            let _ = self.launch_language_server(id);
+            let doc_id = self.new_document(doc);
+            let _ = self.launch_language_servers(doc_id);
 
-            id
+            doc_id
```

Comment:
seems like an unrelated change that slipped trough (renaming `id` to `doc_id` ofcourse the method rename makes sense)

Reply:
I guess I thought this was a little bit more clear that `launch_language_servers` launches language servers for the document (it reads better IMHO). But I can revert that change (I don't think a PR itself is worth this change).

Reply:
it's also called `id` in the rest of the function so I would want to stay consistent within the function at least but then you would have to add even more unnecessary changes so I would just drop it from the PR